pangeo-notebook/environment.yml

```
@@ -13,7 +13,7 @@ dependencies:
  - cdsapi
  - cfgrib
  - cf_xarray
- - ciso
+ - ciso 
```

Remove trailing blankspace here.
```suggestion
 - ciso
```